helm/prometheus-operator-app/templates/alertmanager/alertmanager.yaml

```diff
@@ -9,7 +9,11 @@ metadata:
 {{ include "prometheus-operator.labels" . | indent 4 }}
 spec:
 {{- if .Values.alertmanager.alertmanagerSpec.image }}
-  baseImage: {{ .Values.alertmanager.alertmanagerSpec.image.repository }}
+{{- if (.Values.Installation) }}
+  baseImage: "{{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.alertmanager.alertmanagerSpec.image.name }}"
+{{ else }}
+  baseImage: "{{ .Values.image.registry }}/{{ .Values.alertmanager.alertmanagerSpec.image.name }}"
+{{ end }}
```

Comment:
I would suggest to move `image.registry` under `Installation.V1.Registry.Domain` instead of having this condition everywhere. It makes it hard to read the chart.

Reply:
I am not sure we should do that though as this chart can also be used by the app platform

Reply:
Fine with me then, just wanted to make the chart more readable.

Reply:
I agree about the readability. Could we do that using a helper instead so we have that in one place only?